### PR TITLE
Issue 43

### PR DESCRIPTION
**Pull request by RainierBarrett**
Addresses #43. Added doxygen comments to python code in line with HOOMD's style guide.

### CODE REVIEW

**Comment by RainierBarrett** [comment]
If this is good we can close #43. I was just on the same branch as before or something.

**Comment by whitead** [comment]
Are you going to address the pep8 issues?

**Comment by RainierBarrett** [comment]
The problem is that the majority of these are actually the style I found in HOOMD-blue ("too many leading '#' for block comment"). See, for example, [this line](https://github.com/glotzerlab/hoomd-blue/blob/60513d253fe3a32321254644fafee0df5fe03942/hoomd/md/bond.py#L47) in their bond.py file. I can fix the other ones for sure, but should we alter the pep8 style to allow this?

**Comment by whitead** [comment]
Ah, so this could be an issue. Maybe modify `.pep8speaks.yml` to ignore that particular pep8 format? I'll leave it up to you.

File: htf/check_tf_version.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+R""" Check to ensure that we have a compatible version of TensorFlow.
```
Comment (whitead) [inline htf/check_tf_version.py]: Can we use single quotes?

Comment (whitead) [reply htf/check_tf_version.py]: Or is that the hoomd style?

File: htf/graphbuilder.py [hunk 2]
```diff
@@ -38,31 +40,122 @@ def __init__(self, nneighbor_cutoff, output_forces=True):
...
+    # \internal
+    # \brief Stores molecule indices for each atom
+    # \details
+    # Each atom is assigned an index associated with its corresponding molecule
+
+    ## \var mol_batched
+    # \internal
+    # \brief Whether to batch by molecule
+    # \details
+    # Not yet implemented
```
Comment (whitead) [inline htf/graphbuilder.py]: This is implemented now.

File: htf/utils.py [hunk 3]
```diff
@@ -254,7 +281,27 @@ def center_of_mass(positions, mapping, system, name='center-of-mass'):
...
+        Cutoff radius (HOOMD units)
+    NN
+        Maximum number of neighbors per particle
+    system
+        The HOOMD system of particles
+    sorted
+        Whether to sort neighbor lists by distance
+    Returns
+    -------
+    nlist
```
Comment (whitead) [inline htf/utils.py]: In the latest version, it is now N x NN x 4 to match what is done typically.

**Comment by whitead** [comment]
Lots of changes. Nice job figuring out what all of the variables do!

File: htf/check_tf_version.py (hunk 1 above)
Comment (RainierBarrett) [reply htf/check_tf_version.py]: All the HOOMD .py files use double quotes

File: htf/graphbuilder.py (hunk 2 above)
Comment (RainierBarrett) [reply htf/graphbuilder.py]: Can you point me to it? A search on this github repo didn't show me. I searched the master branch.

Comment (whitead) [reply htf/graphbuilder.py]: Here's the PR https://github.com/ur-whitelab/hoomd-tf/pull/47 but it appears I do not use that variable. Could you just remove the variable as part of your PR?

**Pull request merged**